harness/determined/cli/user.py

```diff
@@ -45,9 +45,9 @@
 def list_users(args: Namespace) -> None:
     resp = bindings.get_GetMaster(setup_session(args))
     if resp.to_json().get("rbacEnabled"):
-        render.render_objects(FullUserNoAdmin, client.list_users())
+        render.render_objects(FullUserNoAdmin, client.list_users(active_only=not args.all))
```

Comment:
```
        users_list = client.list_users(active=None if args.all else True)
        renderer = FullUser
        if resp.to_json().get("rbacEnabled"):
            renderer = FullUserNoAdmin
        render.render_objects(renderer, users_list)
```

Reply:
This implementation fails the linter as renderer is being set as two different types (FullUser & FullUserNoAdmin)

Reply:
`renderer = FullUser  # type: Any`?

Reply:
Thanks. Updated!